src/compute-client/src/protocol/command.rs

```diff
@@ -368,14 +369,17 @@ impl Arbitrary for ComputeCommand<mz_repr::Timestamp> {
 /// for anything in this struct.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Arbitrary)]
 pub struct InstanceConfig {
-    /// TODO(database-issues#7533): Add documentation.
+    /// Specification of introspection logging.
     pub logging: LoggingConfig,
+    /// The offset relative to the replica startup at which it should expire. Zero disables feature.
+    pub expiration_offset: Duration,
```

Comment:
I'd feel better if this was an explicit `Option<Duration>`, rather than having the implicit assumption that `ZERO` means "disable". I realize that the assumption existed before, but this seems like a good opportunity to change it?

Reply:
It seems we don't fully support mapping LD flags to options. #29999 for details. Changed the PR to use `Option<Duration>` internally, and convert the duration when reading to an optional that is none if the duration is zero, and some otherwise.